README.md

```diff
@@ -3,8 +3,44 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. <br />
+We are making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
```

Comment:
It _could_ be argued that the word "***empower***" is _over-used_...
see: https://thedevelopmentset.com/17-development-clichés-ill-be-avoiding-in-2017-46c2345a507f